### PR TITLE
fix: 修复 backgroundColorOpacity 不生效问题，close #1201

### LINKED ISSUE #1201: 🐛theme中backgroundOpacity 不生效
### 🖋 Description

修改theme-dataCell-cell-backgroundColorOpacity，但是表格背景没有发生变化rowCell、colCell、cornerCell同
可在https://antv-s2.gitee.io/zh/examples/theme/custom#custom-schema复现

### PR DESCRIPTION
**Pull request by wjgogogo**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [x] Solve the issue and close #1201 

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔍 Self Check before Merge

- [ ] Add or update relevant Docs.
- [ ] Add or update relevant Demos.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/src/facet/header/corner.ts [hunk 1]
```diff
@@ -267,21 +266,4 @@ export class CornerHeader extends BaseHeader<CornerHeaderConfig> {
       },
     });
   }
-
-  protected addBgRect() {
```
Comment (xingwanying) [inline packages/s2-core/src/facet/header/corner.ts:271]: 这个删了树状会不会有问题？

Comment (wjgogogo) [reply packages/s2-core/src/facet/header/corner.ts:271]: 这个完全是冗余操作吧，背景色应该设置在cell里面，而不是header里面

**Review by xingwanying — APPROVED** [review]
LG

**Pull request merged**